lms/services/canvas_api/client.py

```diff
@@ -209,11 +209,14 @@ def post_load(self, data, **_kwargs):  # pylint:disable=no-self-use
             ]
 
     @lru_cache
-    def list_files(self, course_id):
+    def list_files(self, course_id, sort="position"):
         """
         Return the list of files for the given `course_id`.
 
         :param course_id: the Canvas course_id of the course to look in
+        :param sort: field to sort by (on Canvas' API side).
+            Defaults to "position" which is an undocumented option but that it should be the most stable of the options available as it sorts by both "position" and "name" on Canvas' side.
+            https://github.com/instructure/canvas-lms/blob/master/app/controllers/files_controller.rb#L305
```

Comment:
If we're gonna link to a line of the Canvas source code we should probably use a GitHub permalink as line 305 could shift around as the code changes:

```suggestion
            https://github.com/instructure/canvas-lms/blob/d43feb92d40d2c69684c4536f74dec37992c557a/app/controllers/files_controller.rb#L305
```

Reply:
What happens if you send an unknown value to Canvas as the sort parameter? Does the API return an error? Or does it just ignore the parameter? I'm wondering if we'd notice if they removed sort. It sounds like they're only keeping it around for backwards compatibility.

Reply:
> What happens if you send an unknown value to Canvas as the sort parameter? Does the API return an error? Or does it just ignore the parameter? I'm wondering if we'd notice if they removed sort. It sounds like they're only keeping it around for backwards compatibility.

Currently it defaults to sorting by name, the same default as for no parameter at all.